installation/scripts/testing.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-CONCURRENCY=1
+CONCURRENCY=10
```

Comment:
Default CONCURRENCY should stay at `1`. CI plans that allows running tests concurrently will pass `--concurrency` flag to enable it.

Reply:
fixed in latest commit